params.go

```diff
@@ -77,7 +77,7 @@ var (
 	)
 	foragingDeerOutputScaler = flag.Float64(
 		"foragingDeerOutputScaler",
-		18,
+		40,
```

Comment:
default params change? if so should be on separate pr

Reply:
Yeah apologies - should've picked that up in one of the internal PRs. Reverting.